scrapy/utils/reactor.py

```diff
@@ -49,13 +49,20 @@ def __call__(self):
         return self._func(*self._a, **self._kw)
 
 
-def install_reactor(reactor_path):
+def install_reactor(reactor_path, event_loop_path=None):
     """Installs the :mod:`~twisted.internet.reactor` with the specified
-    import path."""
+    import path. Also installs the asyncio event loop with the specified import
+    path if the asyncio reactor is enabled"""
     reactor_class = load_object(reactor_path)
     if reactor_class is asyncioreactor.AsyncioSelectorReactor:
         with suppress(error.ReactorAlreadyInstalledError):
-            asyncioreactor.install(asyncio.get_event_loop())
+            if event_loop_path is not None:
+                event_loop_class = load_object(event_loop_path)
+                event_loop = event_loop_class()
+                asyncio.set_event_loop(event_loop)
+            else:
+                event_loop = asyncio.get_event_loop()
```

Comment:
Is this necessary? Can’t we just skip `asyncioreactor.install` in this scenario altogether?

Reply:
But if we remove this part and `asyncioreactor.install` then how would we deal with the case when no event loop path is specified and the user wants to install the asyncio reactor with the default event loop ?

Reply:
I assume the default event loop is already used by default, if no event loop is installed.

Reply:
I have removed the two lines but have left `asyncioreactor.install` as without it the reactor will not be installed.